docs/event-bus/docs/013-service-programming-model.md

```diff
@@ -0,0 +1,76 @@
+| x-b3-parentspanid | This header is used by the Zipkin tracer in Envoy. The ParentSpanId is 64-bit in length and indicates the position of the parent operation in the trace tree. When the span is the root of the trace tree, the ParentSpanId is absent. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation) e.g. `3f52e7d591bf74e8` |
+| x-b3-sampled | This header is used by the Zipkin tracer in Envoy. When the Sampled flag is either not specified or set to 1, the span will be reported to the tracing system. Once Sampled is set to 0 or 1, the same value should be consistently sent downstream. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `1` |
+| x-b3-spanid | This header is used by the Zipkin tracer in Envoy. The SpanId is 64-bit in length and indicates the position of the current operation in the trace tree. The value should not be interpreted: it may or may not be derived from the value of the TraceId. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `7350c92dc6fa825e` |
+|x-b3-traceid | This header is used by the Zipkin tracer in Envoy. The TraceId is 64-bit in length and indicates the overall ID of the trace. Every span in a trace shares this ID. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). e.g. `02c04beed9aa6c53` |
+| x-request-id | Randomly generated an ID for identifies the HTTP request delivering the business event. e.g. `ec0ffa2a-13b6-9e63-9563-12e5c0e27e38` |
+| x-envoy-decorator-operation | If this header is present on ingress requests, its value will override any locally defined operation (span) name on the server span generated by the tracing mechanism. Similarly, if this header is present on an egress response, its value will override any locally defined operation (span) name on the client span. e.g. `default-route` |
+| x-envoy-expected-rq-timeout-ms | This is the time in milliseconds the router expects the request to be completed. Envoy sets this header so that the upstream host receiving the request can make decisions based on the request timeout, e.g., early exit. This is set on internal requests and is either taken from the `x-envoy-upstream-rq-timeout-ms` header or the route timeout, in that order. e.g. `15000` |
+| x-istio-attributes | istio specific metadata. e.g. `CkMKCnNvdXJjZS51aWQSNRIza3ViZXJuZXRlczovL2NvcmUtcHVzaC01ZDg5OTU1ZjhiLXhjZ3ZyLmt5bWEtc3lzdGVtCh8KCXNvdXJjZS5pcBISMhAAAAAAAAAAAAAA//+sEQAb` |
+
+## Event Payload
+
+The event payload is delivered as the body of the HTTP Request in JSON format. The JSON schema is available in the Service Catalog in the registered service for the remote events.
+
+## Event Payload Example
+
+For example, writing a service interested in an event `order.created` published by `ec` service. the published event is schema is something like:
```

Comment:
Can we refer to the `ec` example in open source?

``In this example, you write a service for an `order.created` Event published by the `ec` service. The published Event is a schema and it looks as follows:``

Reply:
You're right! I've checked and it was advised to change `ec` to `external-solution`

Reply:
Can you please revisit your suggestion to rewrite the `the published event is schema is something like:` as `The published Event is a schema and it looks as follows:`, I believe I had a typo and it should be something like `the published Event/[Event's] schema ...`
The event has a schema and it is not a schema.

Reply:
Side question, why did you favor `looks as` over `looks like`? :)

Reply:
@klaudiagrz please advise